develop/rendering/basic-concepts.md

```diff
@@ -135,7 +135,7 @@ The `MatrixStack` class has the following methods:
 
 You can also multiply the top matrix on the stack using quaternions, which we will cover in the next section.
 
-Taking from our example above, we can make our diamond scale up and down by using the `MatrixStack` and the `tickDelta` - which is the time that has passed since the last frame.
+Taking from our example above, we can make our diamond scale up and down by using the `MatrixStack` and the `tickDelta` - which is the "progress" between the last game tick and the next game tick.
```

Comment:
The example would have to be rewritten or removed entirely. The point is to showcase a framerate-independent method of drawing "animations", which `tickDelta` is incapable of providing

Reply:
If you can provide a better example, I'm happy to update this PR. Otherwise, I can take these changes out and address the rendering stuff separately.

Reply:
Changes reverted.